src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -391,6 +391,144 @@ std::set<std::vector<element::Type>> jit_exp_emitter::get_supported_precisions(c
     return {{element::f32}};
 }
 
+/// GELU_ERF ///
+jit_gelu_erf_emitter::jit_gelu_erf_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                         dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                         const std::shared_ptr<ov::Node>& node)
+        : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+    prepare_table();
+    exp_emitter = std::make_unique<jit_exp_emitter>(h, host_isa, node);
+}
+
+jit_gelu_erf_emitter::jit_gelu_erf_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                         dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                         const ov::element::Type exec_prc) : jit_emitter(host, host_isa, exec_prc) {
+    prepare_table();
+    exp_emitter = std::make_unique<jit_exp_emitter>(h, host_isa, exec_prc);
+}
+
+size_t jit_gelu_erf_emitter::get_inputs_count() const { return 1; }
+
+size_t jit_gelu_erf_emitter::get_aux_vecs_count() const {
+    return exp_emitter->get_aux_vecs_count() + 3;
```

Comment:
If the return value of exp_emitter->get_aux_gprs_count() is independent from the implementation of jit_gelu_erf_emitter, then perhaps we'd better use sth. like:
`return std:max(exp_emitter->get_aux_vecs_count() + k1, k2);`
When exp_emitter->get_aux_vecs_count() is large, we reuse its aux_vecs for the rest of gelu computation. When exp_emitter->get_aux_vecs_count() is small, k2 can guarantee enough aux_vecs for the rest of gelu computation.